code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -506,6 +505,93 @@
 		new /obj/item/stack/sheet/metal(loc, 4)
 	qdel(src)
 
+/**
+  * # Food and Drink Cart
+  *
+  * Secure variant of the [Food Cart][/obj/machinery/smartfridge/foodcart].
+  */
+/obj/machinery/smartfridge/secure/foodcart
+	name = "E.A.T."
+	desc = "Short for the Entrée Armored Transport, this technological marvel by Donk Co. allows Chefs to hawk their food and drink wares around the station, just with a bit more security. Swipe an authorized ID to engage the bluespace-plastitanium anchors."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "foodcart"
+	anchored = FALSE
+	interact_offline = TRUE
+	power_state = NO_POWER_USE
+	visible_contents = FALSE
+	face_while_pulling = FALSE
+	silicon_controllable = FALSE
+
+/obj/machinery/smartfridge/secure/foodcart/Initialize(mapload)
+	. = ..()
+	req_one_access_txt = "[ACCESS_KITCHEN], [ACCESS_BAR]"
```

Comment:
Can you check that this works? `req_one_access_txt` seems to only split `;`, and the accesses might have to have no spaces between